### DISCUSSION TITLE
suggestion: sandbag blocks

### DISCUSSION THREAD

**Discussion by Globalfirestorm10** [HbmMods/Hbm-s-Nuclear-Tech-GIT#555] [General]
I think adding sandbag blocks that could connect like a fence would be cool, it would be good for aa gun nests on bunkers, or decor. It would be good if the devs could add them as a custom block, not just a regular retextured block.

**Comment by yellow8041** [comment]
there actually are! there just not craftable or obtainable other than the laser miner going through fluids

**Reply by Globalfirestorm10** [reply]
im not talking about those blocks. Im talking about something like this: https://www.google.ca/url?sa=i&url=https%3A%2F%2Fwww.123rf.com%2Fphoto_86363894_military-sandbag-artillery-for-army-force-with-cannon.html&psig=AOvVaw34gzWNpRsF8ZUZVp1VBfy6&ust=1653081997287000&source=images&cd=vfe&ved=0CAkQjRxqFwoTCNiogsnA7PcCFQAAAAAdAAAAABAD

Would be cool to have a decor, it would go good with military bases or bunkers

**Comment by Adam29Adam29** [comment]
Sir, this is not a suggestion board and the suggestions will get ignored.

**Reply by Globalfirestorm10** [reply]
there where do I suggest new features?

**Comment by Adam29Adam29** [comment]
Nowhere, as I said suggestions are ignored
